src/utils/history.js

```diff
@@ -0,0 +1,3 @@
+import createBrowserHistory from "history/createBrowserHistory";
```

Comment:
not exactly sure if its needed anymore. Do a research on this one. We now have useLocation and useHistory hooks